### PR TITLE
chore(console): dependency update

### PR DESCRIPTION
**Pull request by peintnermax**
```[tasklist]

### Definition of Ready

- [ ] I am happy with the code
- [ ] Short description of the feature/issue is added in the pr description
- [ ] PR is linked to the corresponding user story
- [ ] Acceptance criteria are met
- [ ] All open todos and follow ups are defined in a new ticket and justified
- [ ] Deviations from the acceptance criteria and design are agreed with the PO and documented.
- [ ] No debug or dead code
- [ ] Critical parts are tested automatically
- [ ] Where possible E2E tests are implemented
- [ ] Documentation/examples are up-to-date
- [ ] All non-functional requirements are met
- [ ] Functionality of the acceptance criteria is checked manually on the dev system.
```

### CODE REVIEW

**Review by eliobischof — DISMISSED** [review]

File: console/src/app/modules/add-key-dialog/add-key-dialog.component.html [hunk 1]
```diff
@@ -16,9 +16,9 @@
     <input cnslInput [matDatepicker]="picker" [min]="startDate" [formControl]="dateControl" />
     <mat-datepicker-toggle style="top: 0" cnslSuffix [for]="picker"></mat-datepicker-toggle>
     <mat-datepicker #picker [startAt]="startDate"></mat-datepicker>
-    <span cnslError *ngIf="dateControl?.errors?.matDatepickerMin?.min">
+    <span cnslError *ngIf="dateControl && dateControl.errors && dateControl.errors['matDatepickerMin']?.min">
```
Comment (eliobischof) [inline console/src/app/modules/add-key-dialog/add-key-dialog.component.html:19]: Can't we still use optional chaining? 

```suggestion
    <span cnslError *ngIf="dateControl?.errors && dateControl.errors['matDatepickerMin']?.min">
```

See other places, too

Comment (peintnermax) [reply console/src/app/modules/add-key-dialog/add-key-dialog.component.html:19]: Bildschirm­foto 2023-04-17 um 10 15 23
this is not accepted by the linter

Comment (eliobischof) [reply console/src/app/modules/add-key-dialog/add-key-dialog.component.html:19]: Also if you insert a question mark after dateConftrol?

Comment (peintnermax) [reply console/src/app/modules/add-key-dialog/add-key-dialog.component.html:19]: Does not do the trick.. Bildschirm­foto 2023-04-18 um 13 50 07

Comment (eliobischof) [reply console/src/app/modules/add-key-dialog/add-key-dialog.component.html:19]: ok, sry

**Review by eliobischof — APPROVED** [review]

**Pull request merged**